webapp/src/components/App/Singlestudy/explore/Debug/index.tsx

```diff
@@ -104,13 +105,15 @@ function Debug() {
 
   return (
     <DebugContext.Provider value={contextValue}>
-      <SplitView id="debug" sizes={[20, 80]} minSize={150}>
+      <SplitView splitId="debug">
         <Box sx={{ p: 1, overflow: "auto", position: "relative" }}>
           <UsePromiseCond
             keepLastResolvedOnReload
             response={treeDataResponse}
             ifPending={() =>
-              Array.from({ length: 3 }).map((_, index) => <Skeleton key={index} height={32} />)
+              Array.from({ length: 3 }).map((_, index) => (
+                <Skeleton key={buildKey(index)} height={32} />
```

Comment:
`key={buildKey(index)}` is the same as `key={index}`. However, there is no issue to use `index` as key for Skeleton.